doc/interop-test-descriptions.md

```diff
@@ -1143,10 +1147,14 @@ Procedures:
       }
     }
     ```
-The call carries a reference to receive the load report, e.g. using CallOptions.
-The reference will be passed to the custom LB policy as part of the `OrcaOobReportListener` API.
-* Client asserts that, after 1.5 second, the latest OOB load report received is equal to the test load report.
-* Client sends another unary call to the server. The call request sets `orca_oob_report` to a 
+The test then blocks until the first response is received. This means the server lock is acquired.
+It is important that if the stream is completed or terminated with error at this time, 
+the test aborts and fails. 
+2. Client verifies (timeout=5s) that the latest OOB load report received is equal to the test load report.
```

Comment:
View the file in github (three dots at the top-right, you get a drop-down, then "View file"). The formatting is pretty broken here. I think you need to indent some of the previous lines, but I'm not certain.